test/unit/news_letter_controller_unit_test.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+class NewsLetterControllerUnitTest < MiniTest::Test
+
+  extend Minitest::Spec::DSL
+  include LogWrapper
+
+  def setup
+    @nl_controller = NewsLetterController.new
+    @mintest_mock1 = MiniTest::Mock.new
+    @mintest_mock2 = MiniTest::Mock.new
+  end
+
+  describe 'Test news-letter email found in google sheets ' do
```

Comment:
what are you testing it for?  would you mind making that a bit clearer?

Reply:
news-letter email is already in google sheets or not

Reply:
news-letter email is already in google sheets or not